### PR TITLE
task 11, done...I think? been on this for so long I don't even know w…

### PR DESCRIPTION
**Pull request by SomeRandomGuy64**
…hat I've written, it just works

### CODE REVIEW

File: __tests__/supertest.test.js [hunk 1]
```diff
@@ -299,5 +297,41 @@ describe('GET /api/users', () => {
...
+            comment_count: expect.any(String),
+          });
+        });
+        expect(reviews).toBeSorted("review_id", {
+          ascending: true,
+        });
+        expect(reviews.length).toBe(11);
+      });
+  });
+  it("invalid category, order or sort_by, returns a 400", () => {
```
Comment (robert112233) [inline __tests__/supertest.test.js:329]: I would separate out these two tests, currently if one is correctly failing then it will cover up the other that is incorrectly passing. 

I would also add a 200 test for a category query that is valid, but has no associated reviews, returning an empty array

**Review by robert112233 — CHANGES_REQUESTED** [review]
Great work, you're pretty much there! Just some tiny feedback below

File: __tests__/supertest.test.js [hunk 2]
```diff
@@ -299,5 +297,66 @@ describe('GET /api/users', () => {
...
+            comment_count: expect.any(String),
+          });
+        });
+        expect(reviews).toBeSorted("review_id", {
+          ascending: true,
+        });
+        expect(reviews.length).toBe(11);
+      });
+  });
+  it("invalid category returns a 400", () => {
```
Comment (RichardNeat) [inline __tests__/supertest.test.js:329]: Technically category not found would be a 404 :)

**Review by RichardNeat — APPROVED** [review]
Nice one! One minor change but consider it approved. Make that change then feel free to merge :)
